Analysis/src/Autocomplete.cpp

```diff
@@ -1275,6 +1292,8 @@ static std::optional<AutocompleteEntryMap> autocompleteStringParams(const Source
         return std::nullopt;
     }
 
+    std::optional<std::string> candidateString = getStringContents(nodes.back());
+
```

Comment:
nit: this is taken very far from the first use, could be moved below the few early exists below.